tests/test-transmuxer.c

```diff
@@ -108,21 +115,98 @@ test_corrupted_empty (void)
   g_unlink (output_file);
 }
 
+static gboolean
+_generate_truncated_file (gchar * src, gchar * dst)
+{
+  gint src_fd, dst_fd, len, i, n;
+  g_autofree gchar *buffer = NULL;
+  g_autoptr (GError) error = NULL;
+
+  src_fd = g_open (src, O_RDONLY);
+
+  if (src_fd < 0)
+    return FALSE;
+
+  len = lseek (src_fd, 0, SEEK_END);
+
+  lseek (src_fd, 0, SEEK_SET);
+
+  // Use len = len / 2 to truncate file in the middle
+  len = len / 2;
+
+  buffer = g_malloc0 (len);
+
+  // Read half the file to have it truncated 
+  n = read (src_fd, buffer, len);
+
+  g_debug ("%d", __LINE__);
```

Comment:
Leftover debug line?

Reply:
It looks like the PR update I did was wrong. Thanks.